### PR TITLE
Create CONTRIBUTING.md for as-proba

### LINKED ISSUE #183: Improve as-proba contribute.md

### PR DESCRIPTION
**Pull request by Elli610**
closes #183

### CODE REVIEW

File: packages/as-proba/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+
+
```
Comment (Ben-Rey) [inline packages/as-proba/CONTRIBUTING.md:2]:
```suggestion
```

**Review by Ben-Rey — APPROVED** [review]

File: packages/as-proba/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Please ensure that your code follows the existing code style used in the project.
+We use the [MassaLabs Prettier configuration](https://github.com/massalabs/prettier-config-as) and [MassaLabs ESLint configuration](https://github.com/massalabs/eslint-config) for formatting and linting.
+
+You can run the following command to format your code before committing:
+
+```sh
+npm run fmt
+```
+
+## Tests
```
Comment (gregLibert) [inline packages/as-proba/CONTRIBUTING.md:46]: You should explain that you are in a mono repo

**Review by gregLibert — CHANGES_REQUESTED** [review]

File: packages/as-proba/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+We are in a monorepo which contains 3 packages (as-types, as-transformer and as-proba). Inside each package is a folder /assembly/__tests__ which contains the test files related to the current package.
```
Comment (gregLibert) [inline packages/as-proba/CONTRIBUTING.md:47]:
```suggestion
```

File: packages/as-proba/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+Please ensure that your changes include any necessary tests.
+We use [as-pect library](https://as-pect.gitbook.io/as-pect/) for unit testing.
+
+You can run the following command to run the tests:
+
+```sh
+npm run test
+```
```
Comment (gregLibert) [inline packages/as-proba/CONTRIBUTING.md:56]:
```suggestion
However, please note that these package tests must also be run from the root of the monorepo when the CI executes npm run test to ensure the package works correctly in the larger context.
```

**Review by gregLibert — CHANGES_REQUESTED** [review]

**Review by gregLibert — APPROVED** [review]

**Pull request merged**